developer/js/source/lexical-model-compiler/lexical-model.ts

```diff
@@ -21,15 +23,58 @@ interface WordBreakerSpec {
    * together at the given strings. e.g.,
    *
    *    joinWordsAt: ['-'] // to keep hyphenated items together
+   *
+   * @since 14.0
    */
   readonly joinWordsAt?: string[];
+
+  /**
+   * Overrides word splitting behaviour for certain scripts.
+   * For example, specifing that spaces break words in certain South-East
+   * Asian scripts that otherwise do not use spaces.
+   *
+   * @since 14.0
+   */
+  readonly overrideScriptDefaults?: OverrideScriptDefaults;
 }
 
 /**
- * Keyman 11.0+ word breaker specification:
+ * Simplified word breaker specification.
+ *
+ * @since 11.0
  */
 type SimpleWordBreakerSpec = 'default' | 'ascii' | WordBreakingFunction;
 
+/**
+ * Override the default word breaking behaviour for some scripts.
+ *
+ * There is currently only one option:
+ *
+ * 'break-words-at-spaces'
+ * : some South-East Asian scripts conventionally do not use spaces to
+ * break words. These scripts are:
+ *
+ *   * Burmese
+ *   * Khmer
+ *   * Thai
+ *   * Laos
+ *
+ * (this list may be incomplete and extended in the future)
+ *
+ * Additionally, some languages that use Katakana exclusively may also use
+ * spaces to separate words.
+ *
+ * For these scripts, the default word breaker breaks at **every**
+ * letter/syllable/ideograph.  Some languages that use these scripts DO use
+ * spaces to separate words. Enable 'break-words-at-spaces' to override the
+ * unwanted, default behaviour for these scripts (breaking at every grapheme
+ * cluster).
```

Comment:
This is a little ambiguous: does `(breaking at every grapheme cluster)` mean the behaviour that is now in play if `break-words-at-spaces` is enabled?

Reply:
Ah, good point! I'll rewrite this!